js/test/unit/builders/utf8-tests.ts

```diff
@@ -47,7 +47,7 @@ function runTestsWithEncoder(name: string, encode: (vals: (string | null)[], nul
     describe(`${encode.name} ${name}`, () => {
         it(`encodes strings no nulls`, async () => {
             const vals = stringsNoNulls(20);
-            validateVector(vals, await encode(vals, []), []);
+            validateVector(vals, await encode([], []), []);
```

Comment:
What's this change about?

Reply:
@trxcllnt I'm not sure about that either and I just noticed it. It seems this change was authored by @handstuyennn before. Would you mind me reverting this line and re-running the tests for another pass?